go-controller/pkg/ovn/controller/services/loadbalancer.go

```diff
@@ -1,4 +1,4 @@
-package loadbalancer
+package services
```

Comment:
so now loadbalancer.go seems like a place where we define our LB struct as well as ensuring and deleting those things from OVN. So create/delete happen here, but get is defined twice in service repair and service controller. Should that be moved here as well?

Reply:
Can you be a little more explicit on what you think we should move here vs combined? I will take a look and try to id them.

Reply:
I meant function func getLBs(nbClient libovsdbclient.Client) ([]*LB, error) and maybe func getServiceLBs(nbClient libovsdbclient.Client) (sets.String, []*LB, error)